src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -353,12 +380,11 @@ function genFragment(
 
   // BR tags
   if (nodeName === 'br') {
-    if (
-      lastLastBlock === 'br' &&
-      (!inBlock ||
-        getBlockTypeForTag(inBlock, lastList, blockRenderMap) === 'unstyled')
-    ) {
-      return {chunk: getBlockDividerChunk('unstyled', depth), entityMap};
+    if (lastLastBlock === 'br' && (!inBlock || inBlockType === 'unstyled')) {
+      return {
+        chunk: getBlockDividerChunk('unstyled', depth, parentKey),
```

Comment:
Only difference here is that we pass the `parentKey`, this is always undefined for root containers but since we call this function recursively when applying it for child-nodes it will be populated with the parent block key